wielder/wield/batch_deploy.py

```diff
@@ -103,6 +103,10 @@ def wield_deployment_batches(conf, action, key_path, func_map):
 
     for batch in batches:
 
+        #TKTK - cannot destroy elk, because filebeat has nothing interesting in the conf
+        if batch == ["elk"] and action == WieldAction.DELETE:
```

Comment:
@pep-tk your fix is hardcoded the best way is to deal with empty configurations or to change the configuration which was probably prematurely merged or left hanging by your truely. I will write a better solution